### DISCUSSION TITLE
Adding OAK camera added to my computer to rtsp server being created and then stream using rtsp link

### DISCUSSION THREAD

**Discussion by Suryansh1089** [bluenviron/mediamtx#1044] [Questions]
Adding OAK camera added to my computer to RTSP server being created and then stream using RTSP link. Can anyone tell me how I can do this?

**Comment by aler9** [comment]
instructions on how to create a stream by using a webcam are available here https://github.com/aler9/rtsp-simple-server#from-a-webcam

[a comment by an automated account is not shown]
